### PR TITLE
Added to Web based file managers

### PR DESCRIPTION
**Pull request by githubber**
But they are more like DAMs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -525,6 +525,8 @@ Some [Groupware](#groupware) solutions also feature file sharing and synchroniza
   * [Apaxy](http://adamwhitcroft.com/apaxy/) - Theme built to enhance the experience of browsing web directories, using the mod_autoindex Apache module and some CSS to override the default style of a directory listing. ([Source Code](https://github.com/AdamWhitcroft/Apaxy) `Public domain` `CSS`
   * [eXtplorer](http://extplorer.sourceforge.net/) - A PHP and JavaScript based File Manager
   * [h5ai](https://larsjung.de/h5ai/) - Modern file indexer for HTTP web servers with focus on your files. Directories are displayed in a appealing way and browsing them is enhanced by different views, a breadcrumb and a tree overview. ([Demo](https://larsjung.de/h5ai/demo/), [Source Code](https://github.com/lrsjng/h5ai)) `MIT` `PHP/Javascript`
+  * [Omeka](http://omeka.org) - Create complex narratives and share rich collections, adhering to Dublin Core standards with Omeka on your server, designed for scholars, museums, libraries, archives, and enthusiasts.. ([Demo](http://omeka.org/showcase/), [Source Code](http://omeka.org/download/)) `GNU GENERAL PUBLIC LICENSE` `Linux/Apache/MySQL/PHP`
```
Comment (nodiscc) [inline README.md]: @githubber 

 * It should be in the https://github.com/Kickball/awesome-selfhosted#content-management-systems-cms section
 * Source code link should be https://github.com/omeka/Omeka (you linked to the download page)
 * License should be written as `GPLv3`
 * Language/platform should simply be `PHP`

File: README.md [hunk 2]
```diff
@@ -525,6 +525,8 @@ Some [Groupware](#groupware) solutions also feature file sharing and synchroniza
...
   * [eXtplorer](http://extplorer.sourceforge.net/) - A PHP and JavaScript based File Manager
   * [h5ai](https://larsjung.de/h5ai/) - Modern file indexer for HTTP web servers with focus on your files. Directories are displayed in a appealing way and browsing them is enhanced by different views, a breadcrumb and a tree overview. ([Demo](https://larsjung.de/h5ai/demo/), [Source Code](https://github.com/lrsjng/h5ai)) `MIT` `PHP/Javascript`
+  * [Omeka](http://omeka.org) - Create complex narratives and share rich collections, adhering to Dublin Core standards with Omeka on your server, designed for scholars, museums, libraries, archives, and enthusiasts.. ([Demo](http://omeka.org/showcase/), [Source Code](http://omeka.org/download/)) `GNU GENERAL PUBLIC LICENSE` `Linux/Apache/MySQL/PHP`  
+  * [ResourceSpace](http://www.resourcespace.org) - ResourceSpace open source digital asset management software is the simple, fast, & free way to organise your digital assets. ([Demo](http://www.resourcespace.org/trial), [Source Code](http://www.resourcespace.org/download)) `BSD-style Open Source license` `MySQL/PHP`
```
Comment (nodiscc) [inline README.md]: * Source code link should point to http://www.resourcespace.org/svn
 * According to http://svn.resourcespace.org/svn/rs/trunk/documentation/licenses/, the license should be `BSD/Other`
 * Language/platform should simply be `PHP`

**Comment by nodiscc** [comment]
Thanks for your contribution. Can you fix the points I mentioned?  
We may consider splitting the section later if it gets too large.

**Comment by githubber** [comment]
Sure, thanks for pointing it out for me.
Shane

**Pull request merged**